src/src.pro

```diff
@@ -1,5 +1,8 @@
 TEMPLATE = app
 
+macx:INCLUDEPATH += /usr/local/opt/libarchive/include
```

Comment:
Wrong place :) Build settings for `libarchive` live in `src/core/core.pri`. And since `homebrew` is only one of the ways to obtain dependencies on OS X, I am not convinced these paths should hardcoded here... It's possible to specify them as `qmake` parameters: `qmake INCLUDEPATH+=... LIBS+=...`. So it might be better be in wiki as well.